build.sbt

```diff
@@ -1,12 +1,21 @@
 import xerial.sbt.Sonatype.GitHubHosting
 import com.lightbend.paradox.markdown.Writer
 
-val playVersion      = "2.8.13"
-val akkaActorVersion = "2.6.15"
-val circeVersion     = "0.14.1"
+val playVersion      = "3.0.0"
+val circeVersion     = "0.14.6"
+val testkitVersion   = "6.0.0"
+val pekko            = {
```

Comment:
Why do we need explicit dependencies on Pekko?

Reply:
different versions are brought transitively, we need to fix them to the same, otherwise pekko throws an exception